packages/components/src/InputText/InputText.test.tsx

```diff
@@ -108,6 +109,39 @@ it("renders a textarea with 4 rows", () => {
   `);
 });
 
+it("renders a regular input with numeric keyboard specified", () => {
+  const tree = renderer
+    .create(<InputText placeholder="Favourite number" keyboard={"numeric"} />)
+    .toJSON();
+  expect(tree).toMatchInlineSnapshot(`
+    <div
+      className="wrapper"
+      style={
+        Object {
+          "--formField-maxLength": undefined,
+        }
+      }
+    >
+      <label
+        className="label"
+        htmlFor="123e4567-e89b-12d3-a456-426655440004"
+      >
+        Favourite number
+      </label>
+      <input
+        className="formField"
+        id="123e4567-e89b-12d3-a456-426655440004"
+        inputMode="numeric"
+        onBlur={[Function]}
+        onChange={[Function]}
+        onFocus={[Function]}
+        onKeyDown={[Function]}
+        type="text"
+      />
+    </div>
+  `);
+});
```

Comment:
Can we write a non-snapshot test for this one?

Something along the lines of:

```
const input = getByLabelText("");
expect(input.attr('inputMode', 'numeric))
```

^^ That is super rough, but I think more clear than a snapshot.

Reply:
Sorry also, maybe this test should be included in `FormField` instead of `InputText` as thats where the prop comes from?

Reply:
yep good call